contrib/cf-remote/cf_remote/commands.py

```diff
@@ -10,12 +10,20 @@ def info(hosts, users=None):
         print_info(data)
 
 
-def install(hub, clients, bootstrap=None, package=None):
+def install(hub, clients, *, bootstrap=None, package=None, hub_package=None, client_package=None):
```

Comment:
Why the `*`?

Reply:
Since they are optional I kinda like to force keywords, so the order doesn't matter. Especially when it's stuff like different types of the same thing (package).